common/app/model/meta/LinkedData.scala

```diff
@@ -0,0 +1,38 @@
+package model.meta
+
+class LinkedData(
+                       val `@type`: String,
+                       val `@context`: String = "http://schema.org")
+
+object LinkedData {
+
+  import org.json4s._
+  import org.json4s.native.Serialization.write
+
+  implicit val formats = DefaultFormats + FieldSerializer[LinkedData]()
+
+  def toJson(list: List[LinkedData]) = write(list)
+}
+
+case class Guardian(
+                     name: String = "The Guardian",
+                     url: String = "http://www.theguardian.com/",
+                     logo: String = "http://assets.guim.co.uk/images/favicons/451963ac2e23633472bf48e2856d3f04/152x152.png",
+                     sameAs: List[String] = List(
```

Comment:
Oh I see that you just copied and paste from the old file, still it would be nice to know who defined that list